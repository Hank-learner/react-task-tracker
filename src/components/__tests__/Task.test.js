import renderer from 'react-test-renderer';
import Task from '../Task';

const validProps = {
    reminder: true,
    text: "testing",
    day:"5th sept 1948"
}


describe('Task snapshot test', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<Task task={validProps}/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});