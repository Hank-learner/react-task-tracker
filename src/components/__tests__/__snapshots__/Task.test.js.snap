// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Task snapshot test renders correctly 1`] = `
<div
  className="task reminder"
  onDoubleClick={[Function]}
>
  <h3>
    testing
    <svg
      fill="currentColor"
      height="1em"
      onClick={[Function]}
      stroke="currentColor"
      strokeWidth="0"
      style={
        Object {
          "color": "red",
          "cursor": "pointer",
        }
      }
      viewBox="0 0 352 512"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M242.72 256l100.07-100.07c12.28-12.28 12.28-32.19 0-44.48l-22.24-22.24c-12.28-12.28-32.19-12.28-44.48 0L176 189.28 75.93 89.21c-12.28-12.28-32.19-12.28-44.48 0L9.21 111.45c-12.28 12.28-12.28 32.19 0 44.48L109.28 256 9.21 356.07c-12.28 12.28-12.28 32.19 0 44.48l22.24 22.24c12.28 12.28 32.2 12.28 44.48 0L176 322.72l100.07 100.07c12.28 12.28 32.2 12.28 44.48 0l22.24-22.24c12.28-12.28 12.28-32.19 0-44.48L242.72 256z"
      />
    </svg>
  </h3>
  <p>
    5th sept 1948
  </p>
</div>
`;
